packages/cfd/src/Containers/cfd-password-modal/icon-type.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { Icon } from '@deriv/components';
+import { CFD_PLATFORMS, routes } from '@deriv/shared';
```

Comment:
Try to use CFD_PLATFORMS from helper file in order to reduce the dependency on shared package.
```suggestion
import { routes } from '@deriv/shared';
import { CFD_PLATFORMS, MARKET_TYPE } from '../../Helpers/cfd-config.ts';
```